### PR TITLE
Updated build scripts to use dotnet

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #14 

@victorhurdugaci

### CODE REVIEW

File: src/NuGetPackageVerifier/project.json [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+  "licenseUrl": "",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Newtonsoft.Json": "6.0.6",
+    "NuGet.Core": "2.8.6",
+    "Mono.Cecil": "0.9.5.4"
+  },
+  "commands": {
```
Comment (victorhurdugaci) [inline src/NuGetPackageVerifier/project.json]: You don't need this anymore

**Comment by victorhurdugaci** [comment]
Looks good

**Comment by ajaybhargavb** [comment]
Updated. @pranavkm

File: src/NuGetPackageVerifier/NuGetPackageVerifier.nuspec [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<metadata>
+		<id>NuGetPackageVerifier</id>
```
Comment (pranavkm) [inline src/NuGetPackageVerifier/NuGetPackageVerifier.nuspec]: Weird indentation. Can you ctrl K - D in VS (with indent set to 2)?

File: src/NuGetPackageVerifier/NuGetPackageVerifier.nuspec [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```
Comment (pranavkm) [inline src/NuGetPackageVerifier/NuGetPackageVerifier.nuspec]: `<package>`

File: makefile.shade [hunk 4]
```diff
@@ -3,8 +3,19 @@ var VERSION='0.1'
...
\ No newline at end of file
+  k-xml-docs-test
+
+#nuget-pack target='package'
+  @{
+  	var outputDir = Path.Combine(TARGET_DIR, "build");
+  	var existingNupkg = Directory.EnumerateFiles(outputDir).First(f => f.EndsWith(".nupkg"));
+  	var version = Path.GetFileNameWithoutExtension(existingNupkg).Substring("NuGetPackageVerifier.".Length);
+  }
+  exec program='.build/NuGet.exe' commandline='pack -nopack ${Path.Combine("src", "NuGetPackageVerifier", "NuGetPackageVerifier.nuspec")} -version ${version} -base ${Path.Combine(TARGET_DIR, "build", "NuGetPackageVerifier", "net451")} -out ${outputDir}'
```
Comment (pranavkm) [inline makefile.shade]: Does KoreBuild allow wrapping these on to a second line? If not, could you extract the paths into variables?

**Comment by pranavkm** [comment]
:watch:

**Comment by victorhurdugaci** [comment]
Agree with @pranavkm comments but otherwise it looks good. Don't forget to change the KoreBuild-dotnet scripts to download and run the new tool

**Comment by ajaybhargavb** [comment]
Updated.

**Comment by pranavkm** [comment]
:shipit:

File: makefile.shade [hunk 5]
```diff
@@ -3,8 +3,21 @@ var VERSION='0.1'
...
 k-standard-goals
 
 #xml-docs-test .clean .build-compile description='Check generated XML documentation files for errors' target='test'
-  k-xml-docs-test
\ No newline at end of file
+  k-xml-docs-test
+
+#nuget-pack target='package'
+  @{
+  	var outputDir = Path.Combine(TARGET_DIR, "build");
```
Comment (victorhurdugaci) [inline makefile.shade]: Question... Is the build output runnable? As far as I know, if you want a self contained app you need to publish it

Comment (ajaybhargavb) [inline makefile.shade]: It is runnable. No need to publish.

**Pull request closed without merging**